packages/nameguard-react/src/components/Report/GraphemeListItem.tsx

```diff
@@ -48,14 +48,13 @@ export function GraphemeListItem({ item }: GraphemeListItemProps) {
               <RiskCounter count={item.risk_count} />
             </div>
 
-            {item.highest_risk && (
```

Comment:
Please double check this change. Thanks

Reply:
https://github.com/namehash/nameguard/pull/256#discussion_r1600551012